### PR TITLE
completed 7

### PR DESCRIPTION
**Pull request by AnnaNorman**

### CODE REVIEW

File: __tests__/test.test.js [hunk 1]
```diff
@@ -133,3 +133,36 @@ test("status:400, responds with an error message when passed a bad article ID",
...
+    .post("/api/articles/1/comments")
+    .send({
+      username: undefined,
+      body: undefined,
+    })
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Missing required fields");
+    });
+});
```
Comment (emilyb93) [inline __tests__/test.test.js:250]: There are some more tests to consider here;
What if we post a valid comment to an article that doesnt yet exist?
What if we post a comment with a username that isnt in our user database?
What if we post a comment to an article_id that isnt a number?

**Review by emilyb93 — CHANGES_REQUESTED** [review]
more testing needed but looking great so far!

**Review by Harrison365 — CHANGES_REQUESTED** [review]
Hello Anna :) 

There are a couple of small issues. you will need to switch back to branch_7, fix them and push again. (then you can move back to 8 whilst we check again).

1) Please return the POSTed article on a key of comment rather than insertedComment.
2) You may need to rename your 404 test to "responds with an error message when passed a bad username" as I believe that the article_id is fine. 

Let us know when you push 7 again :)

**Review by Harrison365 — APPROVED** [review]
All good for #7 :)

**Pull request merged**